eng/Version.Details.xml

```diff
@@ -85,317 +100,316 @@
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.Configuration" Version="3.1.0" Pinned="true">
-      <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>1c5c7777ea9a19d54ab67ec1521665c99460efc5</Sha>
+    <Dependency Name="Microsoft.Extensions.Logging.AzureAppServices" Version="3.1.2"  Pinned="true">
+      <Uri>https://github.com/dotnet/extensions</Uri>
+      <Sha>1286a6ff55e300352dabeb6d778c9fcdd258bd08</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.Console" Version="3.1.0" Pinned="true">
-      <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>1c5c7777ea9a19d54ab67ec1521665c99460efc5</Sha>
+    <Dependency Name="Microsoft.Extensions.Logging.Configuration" Version="3.1.2"  Pinned="true">
+      <Uri>https://github.com/dotnet/extensions</Uri>
+      <Sha>1286a6ff55e300352dabeb6d778c9fcdd258bd08</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.Debug" Version="3.1.0" Pinned="true">
-      <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>1c5c7777ea9a19d54ab67ec1521665c99460efc5</Sha>
+    <Dependency Name="Microsoft.Extensions.Logging.Console" Version="3.1.2"  Pinned="true">
+      <Uri>https://github.com/dotnet/extensions</Uri>
+      <Sha>1286a6ff55e300352dabeb6d778c9fcdd258bd08</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.EventSource" Version="3.1.0" Pinned="true">
-      <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>1c5c7777ea9a19d54ab67ec1521665c99460efc5</Sha>
+    <Dependency Name="Microsoft.Extensions.Logging.Debug" Version="3.1.2"  Pinned="true">
+      <Uri>https://github.com/dotnet/extensions</Uri>
+      <Sha>1286a6ff55e300352dabeb6d778c9fcdd258bd08</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.EventLog" Version="3.1.0" Pinned="true">
-      <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>1c5c7777ea9a19d54ab67ec1521665c99460efc5</Sha>
+    <Dependency Name="Microsoft.Extensions.Logging.EventSource" Version="3.1.2"  Pinned="true">
+      <Uri>https://github.com/dotnet/extensions</Uri>
+      <Sha>1286a6ff55e300352dabeb6d778c9fcdd258bd08</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.TraceSource" Version="3.1.0" Pinned="true">
-      <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>1c5c7777ea9a19d54ab67ec1521665c99460efc5</Sha>
+    <Dependency Name="Microsoft.Extensions.Logging.EventLog" Version="3.1.2"  Pinned="true">
+      <Uri>https://github.com/dotnet/extensions</Uri>
+      <Sha>1286a6ff55e300352dabeb6d778c9fcdd258bd08</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.Testing" Version="3.1.0-rtm.19572.8" CoherentParentDependency="Microsoft.EntityFrameworkCore">
-      <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>1c5c7777ea9a19d54ab67ec1521665c99460efc5</Sha>
+    <Dependency Name="Microsoft.Extensions.Logging.TraceSource" Version="3.1.2"  Pinned="true">
```

Comment:
You don't have a matching `</Dependency>` for this one and it's breaking on the internal builds